examples/python/v2/cli/find_did.py

```diff
@@ -1,11 +1,13 @@
 from sock import get
 
 
-def main():
-    endpoint = "/identifiers/did:nodex:test:EiD_ZSrS4E4FZruAIJnMt1KjvH1HvwCRYdnIzYpQr4vsuQ"
-    json_response = get(endpoint)
-    print(json_response)
+# PLEASE WRITE destination_did
+destination_did = (
+    "did:nodex:test:DummyDummyDummyDummyDummyDummyDummyDummyDummyD"
+)
 
+endpoint = "/identifiers/" + destination_did
+json_response = get(endpoint)
```

Comment:
```suggestion
json_response = get(f"/identifiers/{destination_did}")
```
`endpoint` does not need to be defined for consistency with other examples?